Docs/landing/data/releases.toml

```diff
@@ -1,7 +1,12 @@
-current = "2.7.0"
+current = "2.8.0"
 [[versions]]
-  version = "2.7.0"
+  version = "2.8.0"
   status = "current"
+  docs = "./2.8/"
```

Comment:
I can't fully explain why we sometimes use 3-part version numbers and sometimes use 2-part version numbers.

I just followed what I saw.

Reply:
I think so the URL's don't change with every patch release